e2e/cypress/e2e/organization/organizations.cy.ts

```diff
@@ -1,4 +1,40 @@
+import { ensureOrgExists, renameOrg } from 'support/api/orgs';
+import { apiAuth } from '../../support/api/apiauth';
+
+const orgListPath = `/orgs`;
+const orgPath = `/org`;
+
+const orgNameOnCreation = 'e2eneworg';
+const testOrgNameChange = 'e2erenametest';
+
 describe('organizations', () => {
+  describe('rename', () => {
+    beforeEach(() => {
+      apiAuth()
+        .as('api')
+        .then((api) => {
+          ensureOrgExists(api, orgNameOnCreation)
+            .as('newOrgId')
+            .then((newOrgId) => {
+              cy.visit(orgListPath);
```

Comment:
ensure org has name?
doesn't the test fail if you run it twice?

Reply:
it ensures that it exists..

Reply:
if it runs twice and the org was already renamed, a new org should be created

Reply:
process now its like this: beforeEach -> ensureorg exists with fixed name, rename to unique name